service/src/test/java/bio/terra/catalog/datarepo/DatarepoServiceTest.java

```diff
@@ -165,4 +166,13 @@ void getPreviewTableDatarepoException() throws Exception {
     assertThat(t.getStatusCode(), is(HttpStatus.NOT_FOUND));
     assertThat(t.getMessage(), is("bio.terra.datarepo.client.ApiException: " + errorMessage));
   }
+
+  @Test
+  void getExportDatarepoException() {
```

Comment:
This is fine although it's only needed because this PR adds code to `DatasetService`. Alternatively you could make TDR snapshots the same as external (no-op in the switch) for now.